Dockerfile

```diff
@@ -1,16 +1,23 @@
-FROM andreiungur/ecse437:cmake-base
+FROM andreiungur/ecse437:cmake-base2
 
 WORKDIR /neutralinojs
 COPY . /neutralinojs
-CMD cmake .
+CMD cd build && cmake -DCMAKE_C_COMPILER=/usr/bin/gcc-8 -DCMAKE_CXX_COMPILER=/usr/bin/g++-8 /neutralinojs && make
 
 # Sets up base docker image
```

Comment:
Maybe update this comment to say that the commands below are what's included in the `andreiungur/ecse437:cmake-base2` base image

Reply:
Good call, done!